app/services/formbuilder-id-service.js

```diff
@@ -0,0 +1,19 @@
+import Service from '@ember/service';
+
+const EMPTY_FORMBUILDER_ID = '';
+
+export default class FormbuilderIdServiceService extends Service {
```

Comment:
Not a fan of the service name tbh..

* formbuilder => the application
* id => managing the application instance id's? (blowing things up 😁 )
* service is twice in the name

Did you check the sticky router params that @aatauil mentioned?

A service is a little to big for just going back one route in one place IMO if the sticky query router param does not work. Not sure if this is something but we can maybe store it in the storage of the browser? @aatauil

Reply:
yes will check on that